### ISSUE TITLE
V3 File Upload - File is not being uploaded consistently

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/vets-design-system-documentation#3106]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Select a file to upload, the file uploads but receive the following message: Upload a completed [Form Number].  If you refresh your screen and try again the file uploads.  This does not happen all of the time. 
<img width="619" alt="2024-07-30_12-57-44" src="https://github.com/user-attachments/assets/5d6d2bf0-08db-4c97-9c72-e81d3d07d87b">

- Note:  During user research this happened to at least 4 participants.
-
## What I expected to happen

<!--
  What did you expect to happen?
- File would upload and I could click continue

## Reproducing

- Formation version:
- Device:Mackbook
- Browser: Chrome 126

Steps to reproduce:

1. Select choose from folder
2. Select a document to upload
3.
4.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [X] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/3106?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @powellkerry @rmessina1010 @rsmithadhoc

**Comment by harshil1793** [comment]
I have communicated with the team regarding the issue with their instance, specifically related to the `errorMessages` prop in `upload.jsx` and `fileInputPattern.jsx`. After testing by removing the `errorMessages` prop, I confirmed that no errors occurred, indicating that the `errorMessages` should be made conditional. The default state should be `null`, and the `errorMessages` should only be set when there are issues with the file.

Given that the team may need further assistance in implementing this conditional logic or addressing related issues, I recommend keeping this ticket open. This will allow us to provide any necessary support as they work on resolving the issue.
